src/main/java/com/google/alpollo/AnalysisHelper.java

```diff
@@ -0,0 +1,29 @@
+package com.google.alpollo;
+
+import java.io.IOException;
+import com.google.api.gax.rpc.FixedHeaderProvider;
+import com.google.cloud.language.v1.Document;
+import com.google.cloud.language.v1.LanguageServiceClient;
+import com.google.cloud.language.v1.LanguageServiceSettings;
+import com.google.cloud.language.v1.Sentiment;
+
+/**
+ * Helper class with methods that use the Natural Language API or use the data that comes
+ * from the API
+ */
+public class AnalysisHelper {
+  /**
+   * Based on the lyrics sent, the AI can extract the main "sentiment" of the text.
+   * This sentiment has a score, showing the overall positivity of the text, ranging from -1.0 to 1.0
+   * and a magnitude, representing how strong the sentiment is, ranging from 0.0 to +inf
+   */
+  public Sentiment getSentiment(String lyrics) throws IOException {
+    LanguageServiceSettings settings = LanguageServiceSettings.newBuilder().setHeaderProvider(
+        FixedHeaderProvider.create("X-Goog-User-Project", "google.com:alpollo-step-2020")).build();
```

Comment:
Does this need to be re-created every time you use it, or could it be stored?

Reply:
You're right, it would be better to store it, but when I tried to store it outside the method I got an error message like `Default constructor cannot handle exception type IOException`, so I left it like this until now.

Reply:
I would have to handle the exception somehow at class-level?

Reply:
You could have a static field to store this. In the method, you check whether the field is null. If it is not null, use the value stored in it. Otherwise create a new LanguageServiceSettings and store the value in the filed for the future.